salt/modules/cohesity.py

```diff
@@ -0,0 +1,522 @@
+"""
+This module have been tested on Cohesity API v1.
+:depends: cohesity-management-sdk,
+https://github.com/cohesity/management-sdk-python
+sdk can be installed using `pip install cohesity-management-sdk`
+"""
+
+# Python Modules Import
+import copy
+import logging
+import os
+
+try:
+    import salt.utils.files
+
+    from cohesity_management_sdk.cohesity_client import CohesityClient
+    from cohesity_management_sdk.exceptions.api_exception import APIException
+    from cohesity_management_sdk.models.cancel_protection_job_run_param import (
+        CancelProtectionJobRunParam,
+    )
+    from cohesity_management_sdk.models.change_protection_job_state_param import (
+        ChangeProtectionJobStateParam,
+    )
+    from cohesity_management_sdk.models.delete_protection_job_param import (
+        DeleteProtectionJobParam,
+    )
+    from cohesity_management_sdk.models.environment_register_protection_source_parameters_enum import (
+        EnvironmentRegisterProtectionSourceParametersEnum as env_enum,
+    )
+    from cohesity_management_sdk.models.protection_job_request_body import (
+        ProtectionJobRequestBody,
+    )
+    from cohesity_management_sdk.models.recover_task_request import RecoverTaskRequest
+    from cohesity_management_sdk.models.register_protection_source_parameters import (
+        RegisterProtectionSourceParameters,
+    )
+    from cohesity_management_sdk.models.restore_object_details import (
+        RestoreObjectDetails,
+    )
+    from cohesity_management_sdk.models.run_protection_job_param import (
+        RunProtectionJobParam,
+    )
+    from cohesity_management_sdk.models.universal_id import UniversalId
+    from cohesity_management_sdk.models.update_protection_jobs_state_request_body import (
+        UpdateProtectionJobsStateRequestBody,
+    )
+    from cohesity_management_sdk.models.vmware_restore_parameters import (
+        VmwareRestoreParameters,
+    )
+
+    HAS_LIBS = True
+except ImportError as err:
+    HAS_LIBS = False
+    print("Error while importing Cohesity SDK modules.")
+    print(err)
+    exit(0)
+
+logger = logging.getLogger(__name__)
```

Comment:
```suggestion
log = logging.getLogger(__name__)
```
Please use `log` and not `logger` since that's what's used throughout Salt's codebase.

Reply:
Done